patches/server/0003-Leaf-Config.patch

```diff
@@ -81,24 +88,30 @@ index 0000000000000000000000000000000000000000..f05e1ee109a36c39857f75c296f2ff21
 +            module.onLoaded();
 +
 +            modules.add(module);
++            for (Field field : getAnnotatedStaticFields(clazz, Experimental.class)) {
++                Object obj = field.get(null);
++                if (!(obj instanceof Boolean)) continue;
++                boolean enabled = (Boolean) obj;
++                if (enabled) {
++                    LeafConfig.LOGGER.warn("Experimental feature {} is enabled, please report any bugs you found!", clazz.getSimpleName());
```

Comment:
It's better to collect those exp feature names, and warn after the loop, to prevent console spam